packages/augur-core/source/tools/installSolidityFlattener

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
The python docker image doesn't come with bash any more, any we didn't need it, anyway